CRM/Admin/Form/ScheduleReminders.php

```diff
@@ -404,10 +409,6 @@ public function setDefaultValues() {
       else {
         $defaults['entity'] = $entityStatus;
       }
-      if ($absoluteDate = CRM_Utils_Array::value('absolute_date', $defaults)) {
```

Comment:
@eileenmcnaughton this would be where if we had to do any parsing we would i guess.

Reply:
yeah makes sense - if it's already being stored sensibly we are probably ok